### ISSUE TITLE
preview html view for mat-editor

### ISSUE THREAD

**Issue by anjunar** [anjunar/blomst-parent#22]

**Comment by anjunar** [comment]
In welcher weise soll den der HTML preview dargestellt werden? Ein neues Fenster indem das HTML angezeigt wird?

**Comment by FireFoxX79** [comment]
Ich mag die Funktion wie auf Github in einem Zusätzlichen Tab.

**Comment by anjunar** [comment]
Working in version 1.02 alpha

**Comment by FireFoxX79** [comment]
Die Preview im Editor ist klasse. Ich bin momentan noch nicht sicher ob ich es gut finde, das du die Funktionen im Write Modus in Seperate Tabs gepackt hast. Zumindest wenn ich einen "großen" Bildschirm habe sind sie nicht nötig. Auf einem Handydisplay mögen Sie Sinn ergeben.

**Comment by anjunar** [comment]
Genau, ich habe da auch an die Version auf dem Handy gedacht. Dann haben die Funktionen auch alle Platz auf dem Display.

Vielleicht bekommen die Posts auch einen Editor. Und das Fenster für die Posts hat nur eine Breite von 400px bis 600px.
